Cargo.toml

```diff
@@ -33,11 +31,10 @@ async-trait = "0.1"
 once_cell = "1"
 assert-json-diff = "2.0.1"
 base64 = "0.21.0"
+url = "2.2"
 
 [dev-dependencies]
-async-std = { version = "1.9.0", features = ["attributes"] }
-surf = "2.3.2"
 reqwest = "0.11.3"
-tokio = { version = "1.5.0", features = ["macros", "rt-multi-thread"] }
+tokio = { version = "1.5.0", features = ["macros", "rt-multi-thread", "test-util"] }
+serde = { version = "1", features = ["derive"] }
 actix-rt = "2.2.0"
-isahc = "1.3.1"
```

Comment:
We want to keep testing that we work on all these runtimes.

Reply:
The test suite shouldn't be touched beyond the changes forced by API differences.

Reply:
a16eb6ee05fadc218f36c2a9821458a1c5ae2600 was introduced mainly because I suppose `http-rs/surf` is not active mantained as `http-rs/http-types`?

Re: we work on all these runtimes/HTTP clients
I don't think it's a problem, because we use `hyper` as a mock server and `http` as transport protocol, which is quite standard and all HTTP clients should obey.

It's better to review commit by commit. The first commit did most of the work except for doc tests, and the remaining commits are housekeeping.

Reply:
There are sharp edges in how things interoperate across runtimes, which is why those tests were there in the first place using those specific libs. There is no problem in keeping a not-very-actively-maintained library as a dev dependency.